spec/services/request_create_service_spec.rb

```diff
@@ -12,6 +12,20 @@
     end
   end
 
+  context 'with auto fill requester' do
+    it 'auto fill requester if it is nil' do
+      request = subject.create(:name => 'req1', :content => 'test me')
+      request.reload
+      expect(request.requester).to eq("John Doe")
```

Comment:
same, get the full name from current header

Reply:
There is no `fullname` method in in either `header` or `user`